packages/inter-protocol/src/price/fluxAggregator.contract.js

```diff
@@ -0,0 +1,63 @@
+import { AssetKind, makeIssuerKit } from '@agoric/ertp';
+import { assertAllDefined } from '@agoric/internal';
+import { E } from '@endo/eventual-send';
+import { provideFluxAggregator } from './fluxAggregator.js';
+
+/**
+ * @typedef {import('@agoric/vat-data').Baggage} Baggage
+ * @typedef {import('@agoric/time/src/types').TimerService} TimerService
+ */
+
+/**
+ * PriceAuthority for their median. Unlike the simpler `priceAggregator.js`, this approximates
+ * the *Node Operator Aggregation* logic of [Chainlink price
+ * feeds](https://blog.chain.link/levels-of-data-aggregation-in-chainlink-price-feeds/).
+ *
+ * @param {ZCF<import('./fluxAggregator.js').ChainlinkConfig & {
+ * timer: TimerService,
+ * brandIn: Brand<'nat'>,
+ * brandOut: Brand<'nat'>,
+ * unitAmountIn?: Amount<'nat'>,
+ * }>} zcf
+ * @param {{
+ * marshaller: Marshaller,
+ * quoteMint?: ERef<Mint<'set'>>,
+ * storageNode: ERef<StorageNode>,
+ * }} privateArgs
+ * @param {Baggage} baggage
+ */
+export const start = async (zcf, privateArgs, baggage) => {
+  const { timer: timerP } = zcf.getTerms();
+
+  const quoteMintP =
+    privateArgs.quoteMint || makeIssuerKit('quote', AssetKind.SET).mint;
+  const [quoteMint, quoteIssuerRecord] = await Promise.all([
+    quoteMintP,
+    zcf.saveIssuer(E(quoteMintP).getIssuer(), 'Quote'),
+  ]);
+  const quoteKit = {
```

Comment:
If the fluxAggregator has a quoteMint, it seems like that should be publicly visible. I see that it eventually makes it way to the priceAuthorities, which presumably share it, but shouldn't it be available from the fluxAggregator as well?

Reply:
> If the fluxAggregator has a quoteMint, it seems like that should be publicly visible.

Why?

> I see that it eventually makes it way to the priceAuthorities, which presumably share it

Not sure where you're seeing that. Here the PA gets just quoteKit.issuer,
```
  const { priceAuthority } = makeOnewayPriceAuthorityKit({
    createQuote: roundsManagerKit.contract.makeCreateQuote(),
    notifier: makeNotifierFromSubscriber(answerSubscriber),
    quoteIssuer: quoteKit.issuer,
    timer: timerPresence,
    actualBrandIn: brandIn,
    actualBrandOut: brandOut,
  });
```
The other read of `quoteKit` in this module is to pass it to `makeRoundsManagerKit`, which uses it only for,
```
        const quoteAmount = AmountMath.make(quoteKit.brand, harden(quote));
        const quotePayment = await E(quoteKit.mint).mintPayment(quoteAmount);
```

Reply:
Sorry, I meant that the issuer should be public. The priceAuthorities provide access to their quoteIssuer (I think). Shouldn't the aggregator do the same? Are there parties to the conversation that know the aggregator and don't talk to individual PAs?  If not, then I guess it's not necessary, but it feels like the Aggregator should be the main authority on the Issuer if it's the one that makes it.

Reply:
> Are there parties to the conversation that know the aggregator and don't talk to individual PAs?

Not on the consumption side, TMK.